### DISCUSSION TITLE
BIPES Pi Pico Servo Blocks Coding Error

### DISCUSSION THREAD

**Discussion by Bertolanatol** [BIPES/BIPES#220] [General]
Hi,

When I use Block coding  in BIPES for  Pi Pico and Servo, it generate this python code :   

from machine import PWM
from machine import Pin
pservo = Pin((25))
servo = PWM(pservo,freq=50)
servo.duty(90)

But It doesn't work. the servo block and the code (line : servo.duty(90)) generate this error :  

Traceback (most recent call last):
  File "<stdin>", line 7, in <module>
AttributeError: 'PWM' object has no attribute 'duty'

What can'I do to use blocks coding with servo and Pi pico without having this error ? Thank's

**Comment by jetannenbaum** [comment]
You should use https://bipes.net.br/pico/ui/ for the Pico.

[image: image.png]
[image: image.png]

**Comment by Bertolanatol** [comment]
Thank's
